README.md

```diff
@@ -17,14 +17,21 @@
 
 ## Usage
 
-> Some usage instructions
+### CRUD API
+To create a new user in the database:
+send POST request to domain/api/users with the user name in the body of the request with the name field (name: username)
```

Comment:
Did this require anything in the body, or does it only need the name in the header?